library/L1_Peripheral/lpc40xx/test/capture_test.cpp

```diff
@@ -0,0 +1,153 @@
+#include <cstdint>
+
+#include "L1_Peripheral/cortex/interrupt.hpp"
+#include "L1_Peripheral/interrupt.hpp"
+#include "L0_Platform/lpc40xx/LPC40xx.h"
+#include "L1_Peripheral/lpc40xx/timer.hpp"
+#include "L1_Peripheral/lpc40xx/capture.hpp"
+#include "L4_Testing/testing_frameworks.hpp"
+#include "utility/enum.hpp"
+#include "utility/status.hpp"
+#include "utility/units.hpp"
+
+
+namespace sjsu::lpc40xx
+{
+EMIT_ALL_METHODS(Capture);
+
+void DummyCallback(void) {}
```

Comment:
Could you either put this in a anonymous namespace so the link name does not class against other tests in the `sjsu::lpc40xx` (think about how many other tests probably have a function for an ISR named DummyCallback). The other option is to create a non-capturing lambda in the TEST_CASE.
```suggestion
namespace {
void DummyCallback(void) {}
}
```